### PR TITLE
Updates copyright year

### LINKED ISSUE #272: Update copyright notices for 2017.
The title says it all.

EDIT: b471620 which is currently only on release updates the splash and about screens, but it needs to be merged to dev and there are other places that still need to be updated.  The properties for the windows builds still show 2016 on the downloadable exe (also when built natively on Windows).

### PR DESCRIPTION
**Pull request by HansHauge**
Fixes #272

### CODE REVIEW

File: qa/pull-tester/test_classes.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2016 The Bitcoin Unlimited developers
+# Copyright (c) 2017 The Bitcoin Unlimited developers
```
Comment (sickpig) [inline qa/pull-tester/test_classes.py]: this should become 2016-2017

**Comment by HansHauge** [comment]
Not sure what's up with Travis, probably just needs to be rebuilt. The only thing changed were code comments.

**Comment by gandrewstone** [comment]
thanks for doing this!  sorry about the merge delay.  it was some weird travis failure...

merged into dev and release branches

**Comment by sickpig** [comment]
@gandrewstone there are a bunch of files that needed a fix. All files with 2016 single copyright year should have become 2016-2017 rather than only 2017, right?

update: scratch that @HansHauge seems to have fixed it

**Comment by HansHauge** [comment]
@gandrewstone  No problem! Now I can add "Bitcoin Unlimited Contributor" to my resume...

**Pull request merged**